engine/crates/engine-v2/schema/src/builder/graph.rs

```diff
@@ -648,10 +658,15 @@ impl<'a> GraphBuilder<'a> {
         resolver_id
     }
 
-    fn push_directives(&mut self, config: &Config, directives: Directives) -> IdRange<TypeSystemDirectiveId> {
+    fn push_directives(
+        &mut self,
+        config: &Config,
+        directives: Directives,
+        graph: &mut FederatedGraphV4,
```

Comment:
doesn't need to be `mut`, no? (not a big deal)

Reply:
Will double check

Reply:
Indeed, remove the mut